test/domains.jl

```diff
@@ -5,15 +5,21 @@ if !("." in LOAD_PATH)
 end
 using ClimaLSM
 using ClimaLSM.Domains:
-    Column, RootDomain, HybridBox, Plane, Point, LSMSingleColumnDomain
+    Column,
+    RootDomain,
+    HybridBox,
+    Plane,
+    Point,
+    LSMSingleColumnDomain,
+    SphericalShell
 using ClimaLSM.Domains: coordinates
 
-FT = Float64
+FT = Float32
```

Comment:
Maybe have TestFloatTypes = (Float32, Float64) and iterate over these in each testset. e.g. [here](https://github.com/CliMA/ClimaCore.jl/blob/23e8c9a7344dd7164bad2fffdd478dcf9f47e0c6/test/DataLayouts/data1d.jl#L8-L11)

Reply:
ok! I also combined some of the domain tests into a single "testset" (all the CC ones, i.e. not the Point yet, and not the LSMDomain) to clean up the printed output